src/nspawn/nspawn.c

```diff
@@ -1104,6 +1105,8 @@ static int parse_argv(int argc, char *argv[]) {
         if (e)
                 arg_container_service_name = e;
 
+        arg_use_cgns = getenv_bool("SYSTEMD_NSPAWN_USE_CGNS");
```

Comment:
`getenv_bool()` might return `-ENXIO`. We should have a  check and set `arg_use_cgns` to zero.
```
if (arg_use_cgns == -ENXIO)
        arg_use_cgns = 0;
```